tests/llm_translation/test_openai_realtime.py

```diff
@@ -0,0 +1,192 @@
+import os
+import sys
+import pytest
+
+sys.path.insert(
+    0, os.path.abspath("../..")
+)  # Adds the parent directory to the system path
+
+import litellm
+from litellm.types.realtime import RealtimeQueryParams
+
+
+@pytest.mark.asyncio
+@pytest.mark.skipif(
+    os.environ.get("OPENAI_API_KEY", None) is None,
+    reason="No OpenAI API key provided",
+)
+async def test_openai_realtime_direct_call_no_intent():
+    """
+    End-to-end test calling the actual OpenAI realtime endpoint via LiteLLM SDK
+    without intent parameter. This should succeed without "Invalid intent" error.
+    Uses real websocket connection to OpenAI.
+    """
+    import websockets
+    import asyncio
+    import json
+    
+    # Create a real websocket client that will connect to OpenAI
+    class RealTimeWebSocketClient:
+        def __init__(self):
+            self.messages_sent = []
+            self.messages_received = []
+            self.websocket = None
+            
+        async def accept(self):
+            # Not needed for client-side websocket
+            pass
+            
+        async def send_text(self, message):
+            self.messages_sent.append(message)
+            
+        async def receive_text(self):
+            # For testing, we'll just wait a bit then close
+            await asyncio.sleep(0.5)
+            # Send a simple session update to simulate real usage
+            if len(self.messages_received) == 0:
+                response = {"type": "session.created", "session": {"id": "test_session"}}
+                self.messages_received.append(response)
+                return json.dumps(response)
+            # Close after first exchange
+            raise websockets.exceptions.ConnectionClosed(None, None)
+            
+        async def close(self, code=1000, reason=""):
+            # Connection will be closed by the realtime handler
+            pass
+            
+        @property
+        def headers(self):
+            return {}
+
+    websocket_client = RealTimeWebSocketClient()
+    
+    # Test with no intent parameter - this should NOT produce "Invalid intent" error
+    try:
+        await litellm._arealtime(
```

Comment:
can we validate the response is as expected ? perhaps check if you got a text field ?

Reply:
added 
<img width="2104" height="1106" alt="image" src="https://github.com/user-attachments/assets/97cb0136-fe6b-4c48-ad75-1ca348b72e10" />